.github/workflows/linux.yml

```diff
@@ -110,7 +142,7 @@ jobs:
         with:
           timeout_minutes: 20
           max_attempts: 2 # https://github.com/servo/servo/issues/30683
-          command: python ./mach test-unit --${{ inputs.profile }}
+          command: python3 ./mach test-unit --${{ inputs.profile }}
```

Comment:
SIDE-NOTE: Installing `python-is-python3` package should fix this in ubuntu.

Reply:
done (57a3b502de74cccd51094de77df4840b339eec32) and deployed ([bab8615c411f8](https://github.com/servo/ci-runners/commit/bab8615c411f8641473a7d3f136dd215cd46df2d))